pkg/backup/backup_test.go

```diff
@@ -53,6 +53,54 @@ import (
 	"github.com/heptio/velero/pkg/volume"
 )
 
+func TestBackedUpItemsMatchesTarballContents(t *testing.T) {
+	// TODO: figure out if this can be replaced with the restmapper
+	// (https://github.com/kubernetes/apimachinery/blob/035e418f1ad9b6da47c4e01906a0cfe32f4ee2e7/pkg/api/meta/restmapper.go)
```

Comment:
Couldn't figure out how to get the k8s client to do this mapping for me, so just hardcoded for now. Looks like the only usable method that converts GVKs to GVRs is `UnsafeGuessKindToResource` (https://github.com/kubernetes/apimachinery/blob/master/pkg/api/meta/restmapper.go#L123), which I am wary to use (although might be ok for this test, though so is hardcoding). Would appreciate any pointers if anyone knows how this works?